core/src/main/java/org/renjin/primitives/text/Text.java

```diff
@@ -203,6 +203,13 @@ public static void bindtextdomain(String domain, String dirname) {
     // translation not yet supported.
   }
 
+  @Internal
+  public static StringVector enc2utf8(StringVector inputVector) {
+    // conversion to UTF8 not yet supported

```

Comment:
It's not quite that "conversion to UTF8 is not supported", but rather the way Renjin's string handling is designed, character vectors _always_ use Unicode encoding. I definitely agree that the NO-OP implementation deserves a comment; it would be good to clarify what's going on here.